incubator/hnc/test/conversion/conversion_test.go

```diff
@@ -220,13 +220,18 @@ spec:
 		FieldShouldNotContain(hierCRD, nsB, hierSingleton, ".status.conditions", "code")
 	})
 
-	It("should convert HNCConfig", func() {
+	It("should convert HNCConfig and remove all enforced types from spec and only show them in the status", func() {
```

Comment:
This is testing the _default_ HNCConfig, right? Maybe change the name to reflect that?

Reply:
yes will do.